minecraft_servers/lendmark/manifest.json

```diff
@@ -18,5 +21,25 @@
        "primary": "#5e8949",
        "background": "#353535",
        "text": "#5e8949"
+    },
+    "gamemodes": {
+    "oneblock": {
+      "name": "Oneblock"
+    },
+    "getdown": {
+      "name": "GetDown"
+    },
+    "borderwars": {
+      "name": "Border Wars"
     }
+  },
+  "location": {
+    "city": "Bratislava",
+    "country": "Slovakia",
+    "country_code": "SK"
+  },
+  "discord": {
+    "server_id": 865900839481901066,
+    "rename_to_minecraft_name": true
```

Comment:
Please change this value to `false` (currently no support for non-partner servers)